docs/index.rst

```diff
@@ -131,16 +143,17 @@ and run tests directly.
 Initial Configuration
 ---------------------
 
-To configure Robottelo, create a file named ``robottelo.properties``. You can
-use the ``robottelo.properties.sample`` file as a starting point. Then, edit the
-configuration file so that at least the following attributes are set::
+To configure Robottelo, multiple template yaml files are present to execute different test cases in Robottelo.
+1. server.yaml : Populate server.yaml with ssh credentials and ssh key path. Then, edit the configuration file so that
+at least the following attributes are set::
+
+    HOSTNAME=[FULLY QUALIFIED DOMAIN NAME OR IP ADDRESS]
```

Comment:
``` suggestion
    HOSTNAMES=[FULLY QUALIFIED DOMAIN NAME OR IP ADDRESS]
```

Reply:
more accurately:
`HOSTNAMES=[LIST OF FULLY QUALIFIED DOMAIN NAMES OR IP ADDRESSES]`